modules/apps/portal-odata/portal-odata-impl/src/main/java/com/liferay/portal/odata/internal/filter/ExpressionVisitorImpl.java

```diff
@@ -227,6 +227,16 @@ public Object visitMethodExpression(
 			return _contains(
 				(EntityField)expressions.get(0), expressions.get(1), _locale);
 		}
+		else if (type == MethodExpression.Type.NOW) {
+			if (!expressions.isEmpty()) {
```

Comment:
Why the expression list must NOT be empty to throw this exception?

Reply:
Because the now() function in OData doesn't support any parameters.

http://docs.oasis-open.org/odata/odata/v4.0/odata-v4.0-part2-url-conventions.html

Reply:
Thank you!